doc/neps/nep-0022-ndarray-duck-typing-overview.rst

```diff
@@ -0,0 +1,340 @@
+==========================================================
+NEP 22: duck typing for NumPy arrays – high level overview
+==========================================================
+
+:Author: Stephan Hoyer <shoyer@google.com>, Nathaniel J. Smith <njs@pobox.com>
+:Status: Draft
+:Type: Informational
+:Created: 2018-03-22
+
+Abstract
+--------
+
+We outline a high-level vision for how NumPy will approach handling
+“duck arrays”. This is an Informational-class NEP; it doesn’t
```

Comment:
A hazard of drafting in Google docs is that apparently every quotation mark ended up curly. That might be worth fixing if you have time :).

Reply:
It's easy enough to fix (simple search/replace), but left them because I figured they're actually more typographically correct anyway...

Reply:
I think doxygen can be configured to render these from straight quotes: http://docutils.sourceforge.net/docs/user/smartquotes.html